gradle/build-complete.gradle

```diff
@@ -11,35 +11,31 @@ if (buildNumber) {
             project.delete(uploadFile)
         }
 
-        OS.current()
-                .conditional()
-                .onUnix {
-                    project.exec {
-                        ignoreExitValue = true
-                        workingDir projectDir
-                        commandLine 'bash', '-c', 'journalctl --since "1 hour ago" 2>&1 > journalctl.log'
-                    }
-                }
-                .onWindows {
-
-                }
-                .onMac {
-
-                }
+        if (OS.current() != OS.WINDOWS) {
```

Comment:
I think we only want this to run on Linux since `journalctl` doesn't exist on OSX.